install.md

```diff
@@ -318,11 +318,18 @@ To add build tags to the make option the `BUILDTAGS` variable must be set, for e
 make BUILDTAGS='seccomp apparmor'
 ```
 
-| Build Tag | Feature                            | Dependency  |
-|-----------|------------------------------------|-------------|
-| seccomp   | syscall filtering                  | libseccomp  |
-| selinux   | selinux process and mount labeling | libselinux  |
-| apparmor  | apparmor profile support           | libapparmor |
+| Build Tag                        | Feature                            | Dependency  |
+|----------------------------------|------------------------------------|-------------|
+| apparmor                         | apparmor support                   | libapparmor |
+| exclude_graphdriver_btrfs        | exclude btrfs                      | libbtrfs    |
+| exclude_graphdriver_devicemapper | exclude device-mapper              | libdm       |
+| libdm_no_deferred_remove         | exclude deferred removal in libdm  | libdm       |
+| ostree                           | ostree support (requires selinux)  | ostree-1    |
+| containers_image_ostree_stub     | exclude ostree                     |             |
+| seccomp                          | syscall filtering                  | libseccomp  |
+| selinux                          | selinux process and mount labeling | libselinux  |
```

Comment:
selinux does NOT require libselinux.  It is native go.
ostree does.

Reply:
Thanks for checking!

Reply:
Updated